### ISSUE TITLE
any pretrain model provide?

### ISSUE THREAD

**Issue by blueardour** [AresGao/shufflenet-caffe#1]
Hi, AresGao

Thanks for share the code on github. I wonder if any pre-trained caffemodel could provide?
